non-oss/pharos_pro/addons/kontena-lens/resources/19-license-enforcer-deployment.yml.erb

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: license-enforcer
+  namespace: kontena-lens
+  labels:
+    app: license-enforcer
+spec:
+  selector:
+    matchLabels:
+      app: license-enforcer
+  template:
+    metadata:
+      labels:
+        app: license-enforcer
+    spec:
+      serviceAccountName: license-enforcer
```

Comment:
I think this needs priority class.. eviction might go south.

Reply:
Yes agree. Priority class added.